conf/log4j2.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<Configuration status="info" name="Nutch" packages="">
+  <Properties>
+    <Property name="hadoop.log.dir">./logs</Property>
```

Comment:
I agree @sebastian-nagel , what action do you want me to take here?

Reply:
I've already tried the change using this "take system property `hadoop.log.dir` if defined otherwise use `./logs` (cf. [Log4j2 - Property Substitution](https://logging.apache.org/log4j/2.x/manual/configuration.html#PropertySubstitution)): it allows to override the log file name and folder via the properties which are also set by bin/nutch from the environment variables. Does the suggested change look good to you?